engine/tests/phpunit/unit/Elgg/ValuesUnitTest.php

```diff
@@ -36,4 +36,24 @@ public function timeProvider() {
 			[new DateTime('+2 days')],
 		];
 	}
+	
+	/**
+	 * @dataProvider emptyProvider
+	 */
+	public function testIsEmpty($value, $expected_result) {
+		$this->assertEquals($expected_result, Values::isEmpty($value));
+	}
+	
+	public function emptyProvider() {
+		return [
+			[0, false],
+			[0.0, false],
+			['0', false],
+			[null, true],
+			[false, true],
+			[[], true],
+			['', true],
+			[new \stdClass(), false],
```

Comment:
Maybe cast to array and check if empty? Not sure...

Reply:
No leaving as is, as in PHP this will also not be empty